libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -657,6 +673,7 @@ bool WaveClip::Paste(double t0, const WaveClip &other)
 
        auto copy = std::make_shared<WaveClip>(other, factory, true);
        copy->ClearSequence(copy->GetSequenceStartTime(), copy->GetPlayStartTime());
+       copy->SetTrimLeft(0);
```

Comment:
Good to see these `SetTrim*` calls for completeness.  Did you see any bad consequences when they were omitted?

Reply:
Yes. Can't remember exactly what, probably overlapping clips.